src/ensembl/src/header/Header.scss

```diff
@@ -61,25 +61,15 @@
   }
 }
 
-.aboutEnsembl {
+.topRight {
   display: flex;
-  align-items: baseline;
-  margin-top: 4px;
-  font-size: 12px;
-  color: $light-blue;
-}
-
-.logotypeAbout {
-  height: 10px;
-  fill: $light-blue;
-  margin-left: 0.5rem;
-}
-
-// For disabling temporarily the about Ensembl link in production
-.aboutEnsemblDisabled {
-  color: $medium-dark-grey;
+  align-items: center;
+  margin-left: 20px;
 
-  .logotypeAbout {
-    fill: $medium-dark-grey;
+  .topRightText {
+    font-size: 13px;
+//    font-weight: $light;
```

Comment:
Don't think we need this comment

Reply:
That is very strange, i dont see on my checkout and i dont have any local changes. I remember removing it. Let me investigate why is it showing on github?

Reply:
Found it, i didnt save the file..doh :)